tensorflow/contrib/README.md

```diff
@@ -21,3 +21,6 @@ they would live in `tensorflow/python/ops/foo_ops.py` and
 `tensorflow/python/kernel_tests/foo_ops_test.py`. In `contrib/`, they are part
 of project `foo`, and their full paths are `contrib/foo/python/ops/foo_ops.py`
 and `contrib/foo/python/kernel_tests/foo_ops_test.py`.
+
+For every function for which a direct equivalent exists outside of contrib, we
```

Comment:
I don't think this is necessarily useful to say in the top-level documentation. This is more of a policy issue.